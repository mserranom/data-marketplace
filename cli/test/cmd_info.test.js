const { run } = require("./cli_runner");
const { username, password } = require("./data/credentials");

test("requires user to be logged in", () => {
  run("logout");
  const result = run("info");
  expect(result.code).toBe(1);
  expect(result.stderr).toBe("user is not logged in\n");
});

test("should return user information", () => {
  run(`login --username ${username} --password ${password}`);
  const result = run("info");
  expect(result.code).toBe(0);
  expect(result.stdout).toMatchSnapshot();
});
