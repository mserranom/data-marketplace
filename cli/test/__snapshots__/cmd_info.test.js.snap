// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return user information 1`] = `
"username: miguel
email: gominolas@gmail.com
"
`;
